docs/guide/ingress/annotations.md

```diff
@@ -79,6 +79,13 @@ By default, Ingresses don't belong to any IngressGroup, and we treat it as a "im
         other Kubernetes users may create/modify their Ingresses to belong to the same IngressGroup, and can thus add more rules or overwrite existing rules with higher priority to the ALB for your Ingress.
 
         We'll add more fine-grained access-control in future versions.
+  
+    !!!warning "Rename behavior"
+        ALB is uniquely identified by it's tag `ingress.k8s.aws/stack`, whose value is the name of IngressGroup.
+
```

Comment:
1. i think we can remove this "There is no way to rename an existing IngressGroup while keeping ALB", since
   * the following text already clarified the recreation behavior.
   * technically there is a way, e.g. by manually change the AWS Tags on the existing provisioned ALB. But i don't want to call this hacky solution out in this document.
2.  `When the `groupName` of IngressGroup for Ingress is changed, the Ingress will be moved to new IngressGroup and supported by ALB for the new IngressGroup. If ALB for the new IngressGroup doesn't exist, a new ALB will be created. If the old IngressGroup no longer contain any Ingresses, ALB for old IngressGroup will be deleted, and deletion protection of ALB will be ignored`.
3. we can remove the "If an IngressGroup has only one Ingress (or IngressGroup is not explicit specified), changing name of IngressGroup will cause recreation of Load Balancer. If an IngressGroup has no Ingress, the corrsponding ALB will be removed. In both cases, deletion protection of ALB will be ignored." given the #2 clarified it.